src/android/build/localnotification.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file contains Original Code and/or Modifications of Original Code
+ * as defined in and that are subject to the Apache License
+ * Version 2.0 (the 'License'). You may not use this file except in
+ * compliance with the License. Please obtain a copy of the License at
+ * http://opensource.org/licenses/Apache-2.0/ and read it before using this
+ * file.
+ *
+ * The Original Code and all software distributed under the License are
+ * distributed on an 'AS IS' basis, WITHOUT WARRANTY OF ANY KIND, EITHER
+ * EXPRESS OR IMPLIED, AND APPLE HEREBY DISCLAIMS ALL SUCH WARRANTIES,
+ * INCLUDING WITHOUT LIMITATION, ANY WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE, QUIET ENJOYMENT OR NON-INFRINGEMENT.
+ * Please see the License for the specific language governing rights and
+ * limitations under the License.
+ */
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven {
+        url "https://maven.google.com"
+    }
+}
+
+if (!project.ext.has('appShortcutBadgerVersion')) {
+    ext.appShortcutBadgerVersion = '1.1.19'
+}
+
+dependencies {
+    implementation "me.leolin:ShortcutBadger:${appShortcutBadgerVersion}@aar"
+    implementation 'com.android.support:support-v4:28.0.0'
```

Comment:
In the `plugin.xml` file there is a reference for the same lib but for the version `26.+`

Reply:
thank you, nice detail.
removed the <framework> tag from the plugin.xml